docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/entrypoints/press-release/press-release.scss

```diff
@@ -36,14 +20,35 @@
 	}
 	&__release-info {
 		@include at-media('tablet') {
-			& > div:first-child {
-				@include u-margin-bottom(1);
+			display: flex;
+			justify-content: space-between;
+		}
+		// Override Document Dates Margin for Press Releases
+		// These dates appear under the title for press releases
+		// As opposed to in the article footer otherwise
+		.cgdp-document-dates {
+			@include u-margin-top(0);
+			@include u-margin-bottom(1);
+			@include at-media('tablet') {
+				@include u-margin-bottom(0);
+			}
+		}
+		// This is the contact info
+		// We target it this way due to the drupal element which
+		// Wraps our media contact with a div of id="block-contact-us-press-release"
+		div:has(.cgdp-media-contact) {
+			white-space: nowrap;
+			@include u-margin-left('auto');
+			.cgdp-media-contact {
+				@include u-margin-left(0);
 				@include at-media('tablet') {
-					@include u-margin-bottom(0);
+					@include u-margin-left(2);
+					@include u-text-align('right');
+				}
+				.cgdp-media-contact__info p {
+					@include u-margin-y(0);
+					line-height: 1.5;
```

Comment:
Can we make this a USWDS token please? Also IDK if 1.5 is one, it is like the 4 or 5 token depending on the font.